rpcd/playbooks/elasticsearch.yml

```diff
@@ -54,6 +56,27 @@
       tags:
         - elasticsearch-pre-install
   roles:
-    - "elasticsearch"
+    - role: "ansible-elasticsearch"
+      es_instance_name: "openstack"
```

Comment:
This is already in group_vars/all/elasticsearch.yml - why duplicate it here?

With regards to all these vars applied to the role task as vars, and the playbook vars: It'd be ideal to have everything in one place, and group_vars is preferred as it allows many layers of overridability. Playbook vars do not provide many options to override.